libs/remix-ui/home-tab/src/lib/components/homeTabGetStarted.tsx

```diff
@@ -59,8 +60,10 @@ function HomeTabGetStarted ({plugin}: HomeTabGetStartedProps) {
   const createWorkspace = async (templateName) => {
     await plugin.appManager.activatePlugin('filePanel')
     const timeStamp = Date.now()
-    await plugin.call('filePanel', 'createWorkspace', templateName + "_" + timeStamp, templateName)
-    await plugin.call('filePanel', 'setWorkspace', templateName + "_" + timeStamp)
+    let templateDisplayName = TEMPLATE_NAMES[templateName]
+    templateDisplayName = await plugin.call('filePanale', 'getAvailableWorkspaceName', templateDisplayName)
```

Comment:
filePanel has an extra e - is this mispelling causing a problem?

Reply:
here is the pr https://github.com/ethereum/remix-project/pull/3568